src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/unit_tests/controllers/api/DocumentControllerTests.kt

```diff
@@ -1,103 +1,85 @@
 package org.vaccineimpact.orderlyweb.tests.unit_tests.controllers.api
 
 import com.nhaarman.mockito_kotlin.*
+import org.junit.After
 import org.junit.Test
-import org.vaccineimpact.orderlyweb.FileSystem
+import org.vaccineimpact.orderlyweb.Files
 import org.vaccineimpact.orderlyweb.controllers.api.DocumentController
 import org.vaccineimpact.orderlyweb.db.Config
 import org.vaccineimpact.orderlyweb.db.DocumentRepository
 import org.vaccineimpact.orderlyweb.models.Document
+import java.io.File
 
 class DocumentControllerTests : ControllerTest()
 {
+    @After
+    fun cleanup()
+    {
+        File("documents").deleteRecursively()
```

Comment:
I think we should be able to avoid having to do this if we put isFile on DocumentDetails and reintroduce the mocked FileSystem.

Reply:
great idea